reactor-core/src/main/java/reactor/core/publisher/Flux.java

```diff
@@ -4029,6 +4029,50 @@ public final Flux<T> concatWith(Publisher<? extends T> other) {
 		return concat(this, other);
 	}
 
+	/**
+	 * If Micrometer's {@code context-propagation} library is on the classpath, this
```

Comment:
Very slight adjustment but it isn't Micrometer's, just happens to exist under the micrometer-metrics org. One option is to call it just the "context-propagation library" and turn it into a hyperlink.